src/ffi/ipc.js

```diff
@@ -235,6 +243,8 @@ const init = () => {
   ipcMain.on('registerSharedMDataDecision', onSharedMDataDecision);
   ipcMain.on('registerOnReqError', onReqError);
   ipcMain.on('skipAuthRequest', skipAuthReq);
+  ipcMain.on('sendRenderer', requestMainResource);
+  ipcMain.on('winstonLogger', winstonLogger);
```

Comment:
We have not implemented winston logging in authenticator, but why to have this IPC listener?

Reply:
@shankar2105 Thank you for catching this. I must have mistakenly merged that winston work into my master branch.